### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by immaryw**
update readme based on node.js instrution

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,118 @@
...
+export FLASK_APP=db_connection_sample.py
+flask run -h 0.0.0.0 -p {your_port_number, e.g. 5678} --reload
 ```
 
+Because this is running on a shared machine everyone cannot use port 5000. Everyone will need to use a unique port otherwise you will get an error that the port is in use.
+
+So we specify the Python file for database connection and also set a port number. You could view the database page of nine entries being served by visiting http://access.engr.oregonstate.edu:5678/db-test while you are VPNed into the OSU network.
+
+
+## Forever
```
Comment (knightsamar) [inline README.md]: Could you please rename this section to "Running the Flask application persistently" and add a link to:

1) http://flask.pocoo.org/docs/1.0/tutorial/deploy/#run-with-a-production-server
2) http://flask.pocoo.org/docs/1.0/deploying/wsgi-standalone/

File: README.md [hunk 2]
```diff
@@ -1,39 +1,118 @@
...
+TBD - SKIPP THIS PART
+
+
+## The Many Flips
+
+TBD - SKIPP THIS PART
```
Comment (knightsamar) [inline README.md]: We can copy paste this section from the NodeJS page if it exists.

**Review by knightsamar — CHANGES_REQUESTED** [review]
Thank you for your work on this Jing! Sorry for the late review. Could you please fix this before I merge it ?

**Comment by immaryw** [comment]
They're updated. @knightsamar

**Review by knightsamar — APPROVED** [review]
Thanks a lot for your work on this, @immaryw!

**Pull request merged**